TombEngine/Objects/Effects/Boss.cpp

```diff
@@ -0,0 +1,321 @@
+#include "framework.h"
+#include "Objects/Effects/Boss.h"
+
+#include "Game/collision/collide_room.h"
+#include "Game/effects/effects.h"
+#include "Game/effects/spark.h"
+#include "Game/items.h"
+#include "Game/misc.h"
+#include "Objects/TR3/Entity/PunaBoss.h"
+#include "Specific/setup.h"
+
+using namespace TEN::Effects::Spark;
+using namespace TEN::Entities::Creatures::TR3;
+
+namespace TEN::Effects::Boss
+{
+	void SpawnShield(const ItemInfo& item, const Vector4& color)
+	{
+		if (!item.TestFlags(BOSSFlag_Object, BOSS_Shield))
```

Comment:
Will this be fixed? `BOSS_` and `BOSSFlag_` prefixes break our coding conventions.

Reply:
fixed.